.gitlab-ci.yml

```diff
@@ -0,0 +1,34 @@
+before_script:
+  - docker info
+  - '[ -d tmp ] || mkdir tmp'
+  - git clone https://github.com/OSC/ondemand-packaging.git tmp/ondemand-packaging
+  - cp /systems/osc_certs/gpg/ondemand/.gpgpass $CI_PROJECT_DIR/tmp/ondemand-packaging/
+  - cp /systems/osc_certs/gpg/ondemand/ondemand.sec $CI_PROJECT_DIR/tmp/ondemand-packaging/
+stages:
+  - build
+  - deploy
+
+rpm-build:
+  stage: build
+  only:
+    - tags
+  script:
+    - ./tmp/ondemand-packaging/build.sh -w $CI_PROJECT_DIR/tmp/work -o $CI_PROJECT_DIR/tmp/output -V $CI_COMMIT_TAG -u $CI_PROJECT_DIR/packaging
+  artifacts:
+    paths:
+      - tmp/output
+    name: "$CI_PROJECT_NAME-$CI_COMMIT_TAG"
+
+
+rpm-deploy:
+  variables:
+    RLS_SCRIPT: "./tmp/ondemand-packaging/release.py"
+    RLS_KEY: "/systems/osc_certs/ssh/ondemand-packaging/id_rsa"
+    RLS_OUTPUT: "./tmp/output/*"
+    SECTION: "latest"
+  stage: deploy
+  only:
+    - tags
+  script:
+    - if [[ "$CI_COMMIT_TAG" =~ .*-.* ]]; then export SECTION=ci; fi
```

Comment:
This if statement looks for hyphens (-), not underscores _ .  That's why we hit that main vs. latest bug when it should have went to the 'ci' section.

Reply:
I haven't read through the code enough to follow this comment, but the convention for tagging repos is now:

for semvar use underscores, not hyphens (v1.0.0, v1.1.0_cr1, v1.1.0_alpha.1) and use hyphens for creating a new rpm build of the same version (v1.0.0-2 would rebuild the v1.0.0 rpm and upload it with a "-2" in the name)

Is your comment still valid if you take into account this convention?

Reply:
Oh yeah it is.

Reply:
Which means this pipeline is going to push a build rpm to /latest/ https://code.osu.edu/ondemand/osc-systemstatus/pipelines/61374

Reply:
Fix that but also `latest` is the wrong section so was really 2 problems here. I've updated the check to look for underscores instead of hypens.